docs/migration-guide/to-16.md

```diff
@@ -27,10 +27,11 @@ You should use the following or higher versions of Node.js:
 
 We changed the resolved object of the Promise returned by `stylelint.lint()` so that:
 
-- the [`output`](../user-guide/node-api.md#output) property now only contains formatted problems
+- a new [`report`](../user-guide/node-api.md#report) property contains the formatted problems
 - a new [`code`](../user-guide/node-api.md#code-1) property contains the autofixed code
+- the [`output`](../user-guide/node-api.md#output) property is now deprecated and will be removed in the next major version.
```

Comment:
suggestion

`…deprecated in favor of the new `code` and `report` properties…`

Reply:
Good catch. Fixed 29af41098 👍🏼